doc/user_guide/marks/text.rst

```diff
@@ -16,28 +113,28 @@ Text Table Heatmap
 
     source = data.cars()
 
-    base = alt.Chart(source).transform_aggregate(
-        num_cars='count()',
-        groupby=['Origin', 'Cylinders']
-    ).encode(
-        alt.X('Cylinders:O', scale=alt.Scale(paddingInner=0)),
-        alt.Y('Origin:O', scale=alt.Scale(paddingInner=0)),
+    base = (
+        alt.Chart(source)
+        .transform_aggregate(num_cars="count()", groupby=["Origin", "Cylinders"])
+        .encode(
+            alt.X("Cylinders:O", scale=alt.Scale(paddingInner=0)),
+            alt.Y("Origin:O", scale=alt.Scale(paddingInner=0)),
+        )
     )
 
     heatmap = base.mark_rect().encode(
-        color=alt.Color('num_cars:Q',
-            scale=alt.Scale(scheme='viridis'),
-            legend=alt.Legend(direction='horizontal')
+        color=alt.Color(
+            "num_cars:Q",
+            scale=alt.Scale(scheme="viridis"),
+            legend=alt.Legend(direction="horizontal"),
```

Comment:
This is a good change and should be kept. This alignment of the parameters is more proper than the old version